js/src/toast.js

```diff
@@ -64,6 +68,8 @@ class Toast extends BaseComponent {
 
     this._config = this._getConfig(config)
     this._timeout = null
+    this._activeMouseInteraction = false
+    this._activeKeyboardInteraction = false
```

Comment:
Does it matter if it is mouse or keyboard interaction instead of 'interaction'? Do you have something in mind?

Reply:
Mouse interactions and keyboard interactions need to be tracked separately. For example, if the user has both their cursor, and focus inside of the toast, then their cursor leaves the toast, we need a mechanism in place that still knows about the focus inside the toast element.

For concrete examples of the scenario I'm referring to, take a look at the tests
- "should not auto hide if focus leaves but cursor remains inside"
- "should not auto hide if cursor leaves but focus remains inside"

If those scenarios are something that should be supported, and you can still think of a different way to accomplish the desired behavior, LMK! If we want just 1 property, we couple make it an object like this:
```javascript
this._activeInteraction = {}

// Later in the file, for example
this._activeInteraction.mouse = true|false
```
Although, this is just a surface change, there is nothing fundamentally different about it.

Reply:
I am not sure of keyboard functionality. How many times do we achieve to catch a toast by tabbing into it?
/Cc @patrickhlauke may be the right person to help

Also, as `_activeMouseInteraction` and `_activeKeyboardInteraction` are also booleans, it seems right to prefix them with `has` (_hasActiveMouseInteraction)

Reply:
The original issue states both cursor and focus interaction should prevent the toast from hiding. If the team decides otherwise then this can be simplified like you where originally suggesting, though I'd recommend against it.

Reply:
yes, makes sense to check for both mouse over and for actual keyboard focus being inside the toast